neqo-transport/src/connection/tests/idle.rs

```diff
@@ -53,6 +53,13 @@ fn test_idle_timeout(client: &mut Connection, server: &mut Connection, timeout:
     assert!(matches!(client.state(), State::Closed(_)));
 }
 
+#[test]
+fn init_rtt_configuration() {
+    const CUSTOM_INIT_RTT: Duration = Duration::from_millis(200);
```

Comment:
Suggest to use a multiple of `INITIAL_RTT` instead of hardcoding a value. (We may change `INITIAL_RTT` at some point.)